be/src/util/json.cpp

```diff
@@ -63,67 +63,167 @@ JsonValue JsonValue::from_string(const Slice& value) {
     return JsonValue(builder.slice());
 }
 
-StatusOr<JsonValue> JsonValue::from_simdjson(simdjson::ondemand::value* value) {
-    namespace sj = simdjson::ondemand;
+namespace so = simdjson::ondemand;
```

Comment:
suggest to use Camel name of  type aliases.

Reply:
namespace should observe lower-case naming rule?